backend/core/models/user.py

```diff
@@ -187,10 +187,10 @@ class User(AbstractUser):
         validators=[validate_avatar],
         null=True,
     )
-    trust_score = models.FloatField(
+    voting_right = models.FloatField(
```

Comment:
I'm not very convinced by this name change. Isn't this field exactly how much we trust that the user is a real user?

I don't know well enough how we go from (certified emails + vouching) -> trust scores -> voting weight -> impact on the aggregation of votes. But, I find that it is meaningful to keep track of "trust scores" through the process.

If this renaming was agreed by @lenhoanglnh and @amatissart , then go ahead :)

Reply:
As of now, I would also find preferable to keep "trust_score" as the field name. It's not clear yet how this score will be used in the ML part, and whether it will be used directly as a voting right or if other processing will be applied.   
So "trust_score" seems to be more explicit of what is currently computed.

Reply:
The name `trust_score` seems more accurate for me to.

Voting right may be the result of several parameters : a confidence that the user is not a robot, and a certain expertise in a particular domain. Trust score seems more related to how much we trust this user is a human.

How about `human_trust` or `human_confidence`?